Entities/User.php

```diff
@@ -583,11 +588,64 @@ public static function beforeUserActionValidation($request)
         return $user;
     }
     //-------------------------------------------------
+    public static function beforeUserLoginValidation($request)
+    {
+        //check if already logged in
+        if (\Auth::check())
+        {
+            \Auth::logout();
+        }
+
+        $inputs = $request->all();
+        $inputs['email'] = trim($inputs['email']);
+
+        $rules = array(
+            'email' => 'required|max:150',
+        );
+        $messages = array(
+            'email.required' => 'The email or username field is required.',
```

Comment:
@wri-vikram-wri231 all validation messages should come from language file and use `trans('vaahcms::messages.inactive_account');`

Reply:
Apply this to everywhere it is needed

Reply:
okay Sir , I will check in every models and controllers